content/docs/functions/json_to_record.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Postgres json_to_record function
+subtitle: convert JSON object to a record
+enableTableOfContents: true
+---
+
+You can use the `json_to_record` function to convert a top-level JSON object into a row, with the type specified by the `AS` clause. 
+
+This function is useful when you need to parse JSON data received from external sources, such as APIs or file uploads, and store it in a structured format. By using `json_to_record`, you can easily extract values from JSON and map them to the corresponding columns in your database table. 
+
+**Function Signature**
```

Comment:
```suggestion
**Function signature**
```

Typically we use [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case) capitalization for headings.